### ISSUE TITLE
How difficult would it be to add a third handle for "playing" the ticks in between range?

### ISSUE THREAD

**Issue by dereklin** [angular-slider/angularjs-slider#393]
![third-handle](https://cloud.githubusercontent.com/assets/2231799/17537675/873bcdbc-5e53-11e6-8eaa-dad44c088624.PNG)

Right now the slider has a left and a right handle for defining the range.  I want to put a vertical bar within the range and would like to move the vertical bar along the invisible ticks in between range.  I want to be able to move the vertical bar both manually and programmatically.

Basically, I want to combine the slider with a range Input.

Could you please point me to the right direction?

**Comment by ValentinH** [comment]
This is not feasible with this library and will not be. Indeed, all the logic of the slider is based on having 2 handles. A total rewrite would be needed to handle more handles...
